[2025-03-12T20:33:02.044+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-12T20:33:02.062+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.plot_github scheduled__2025-03-11T13:30:00+00:00 [queued]>
[2025-03-12T20:33:02.070+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.plot_github scheduled__2025-03-11T13:30:00+00:00 [queued]>
[2025-03-12T20:33:02.070+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-03-12T20:33:02.080+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): plot_github> on 2025-03-11 13:30:00+00:00
[2025-03-12T20:33:02.091+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=583) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-12T20:33:02.095+0000] {standard_task_runner.py:64} INFO - Started process 611 to run task
[2025-03-12T20:33:02.098+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks', 'plot_github', 'scheduled__2025-03-11T13:30:00+00:00', '--job-id', '1637', '--raw', '--subdir', 'DAGS_FOLDER/Stock_pred.py', '--cfg-path', '/tmp/tmplkp7jgcd']
[2025-03-12T20:33:02.099+0000] {standard_task_runner.py:91} INFO - Job 1637: Subtask plot_github
[2025-03-12T20:33:02.130+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks.plot_github scheduled__2025-03-11T13:30:00+00:00 [running]> on host b7aba691e1ff
[2025-03-12T20:33:02.174+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks' AIRFLOW_CTX_TASK_ID='plot_github' AIRFLOW_CTX_EXECUTION_DATE='2025-03-11T13:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-11T13:30:00+00:00'
[2025-03-12T20:33:02.175+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-12T20:33:09.997+0000] {logging_mixin.py:188} INFO - images/thumbs/01.jpg UPDATED
[2025-03-12T20:33:17.134+0000] {logging_mixin.py:188} INFO - images/fulls/01.jpg UPDATED
[2025-03-12T20:33:17.135+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-12T20:33:17.135+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-12T20:33:17.139+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks, task_id=plot_github, run_id=scheduled__2025-03-11T13:30:00+00:00, execution_date=20250311T133000, start_date=20250312T203302, end_date=20250312T203317
[2025-03-12T20:33:17.174+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-03-12T20:33:17.183+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-12T20:33:17.183+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
